next.config.js

```diff
@@ -1,17 +1,21 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {}
 
 // next.config.js
 const nextConfiguration = {
-  ...nextConfig,
-  webpack(config) {
-    config.module.rules.push({
-      test: /\.svg$/,
-      use: ["@svgr/webpack"],
-    });
-
-    return config;
-  },
-};
-
-module.exports = nextConfiguration;
+    ...nextConfig,
+    webpack(config) {
+      config.module.rules.push({
+        test: /\.svg$/,
+        use: ["@svgr/webpack"]
+      });
+  
+      return config;
+    },
+    images: {
+      domains: ["s3-alpha-sig.figma.com"]
```

Comment:
The images.domains configuration is deprecated. Please use images.remotePatterns configuration instead, here is the link for the [docs ](https://nextjs.org/docs/app/building-your-application/optimizing/images#remote-images)